tutorials/3d/spring_arm.rst

```diff
@@ -0,0 +1,125 @@
+:article_outdated: False
+
+.. _doc_spring_arm:
+
+Third-person camera with spring arm
+===================================
+
+Introduction
+------------
+
+3D games will often have a third-person camera that follows and
+rotates around something such as a player character or a vehicle.
+
+In Godot, this can be done by setting a :ref:`Camera3D <class_Camera3D>` as a child of a node.
+However, if you try this without any extra steps you'll notice that the camera clips through geometry and hides the scene.
```

Comment:
```suggestion
However, if you try this without any extra steps, you'll notice that the camera clips through geometry and hides the scene.
```

Reply:
Applied in https://github.com/godotengine/godot-docs/commit/e95419499130b61f5d196d64d7e413795c2e736c.